kubernetes/samples/scripts/domain-lifecycle/clusterStatus.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+set -eu
+set -o pipefail
+
+function usage() {
+cat<<EOF
+
+Usage:
+  $(basename $0) [-n DomainNS] [-d DomainUID] [-c clusterName] [-m kubernetesCli] [-h]
+
+  This script dumps a table of WebLogic Cluster status
```

Comment:
As per @rosemarymarano suggestion for the ".md" doc, replace occurrences of "dump" with "display" throughout.